components/builder-depot/src/error.rs

```diff
@@ -163,3 +181,15 @@ impl From<hab_net::NetError> for Error {
         Error::NetError(err)
     }
 }
+
+impl From<rusoto_s3::ListBucketsError> for Error {
```

Comment:
A while ago it was decided we'd stop doing the From's for errors and use .map_err instead.. so we could just remove these.

Reply:
Oh! I didn't know that. Will get these removed as well.

Reply:
I haven't yet cleaned these out, will do this in the morning as well.